src/ol/interaction/translate.js

```diff
@@ -234,6 +240,35 @@ ol.interaction.Translate.prototype.setHitTolerance = function(hitTolerance) {
 
 
 /**
+ * @inheritDoc
+ */
+ol.interaction.Translate.prototype.setMap = function(map) {
+  var oldMap = this.getMap();
+  ol.interaction.Pointer.prototype.setMap.call(this, map);
+  this.updateState_(oldMap);
+};
+
+
+/**
+ * @param {ol.Map} oldMap Old map.
+ * @private
+ */
+ol.interaction.Translate.prototype.updateState_ = function(oldMap) {
```

Comment:
It looks like `oldMap` will be an `ol.Object.Event` when the `active` property is changed.